ballerina/modules/azure.openai/llm_client_azure_open_ai.bal

```diff
@@ -42,9 +44,12 @@ public isolated distinct client class AzureOpenAIModel {
         self.apiVersion = apiVersion;
     }
 
-    isolated remote function call(string prompt, map<json> expectedResponseSchema) returns json|error {
+    isolated remote function call(np:Prompt prompt, typedesc<anydata> expectedResponseTypedesc) returns anydata|error {
```

Comment:
If we add this as a record or a union of `np:Prompt|record{}`, it will not break in future

Reply:
Why `record {}`? What do we anticipate would go in it?